src/current/v25.2/cockroachdb-feature-availability.md

```diff
@@ -279,6 +279,12 @@ Command                                     | Description
 
 For more information, see [Architecture > Replication Layer > Leader leases]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leader-leases).
 
+### Buffered Writes
+
+Buffered Writes enhance transaction throughput and reduce operational cost by minimizing the number of round-trips between the client and the database during write operations.
```

Comment:
This is another place where we may want to say "transaction coordinator" rather than "client"

Reply:
updated this to say "gateway node and other nodes" like in some other places. I don't think most users know about "transaction coordinators" who haven't pored over our architecture docs, but we do expose the gateway node concept already in a buncha places